app/gui/src/controller/ide/desktop.rs

```diff
@@ -39,7 +39,7 @@ const UNNAMED_PROJECT_NAME: &str = "Unnamed";
 #[derivative(Debug)]
 pub struct Handle {
     logger:               Logger,
-    current_project:      Rc<CloneRefCell<model::Project>>,
+    current_project:      Rc<RefCell<Option<model::Project>>>,
```

Comment:
Why this change?

Reply:
Changed it to `CloneCell`, it fits nicer indeed.